server/ast/expr.go

```diff
@@ -222,7 +222,7 @@ func nodeExpr(node tree.Expr) (vitess.Expr, error) {
 		case tree.CastExplicit, tree.CastShort:
 			// Both of these are acceptable
 		case tree.CastPrepend:
-			return nil, fmt.Errorf("typed literals are not yet supported")
+			//return nil, fmt.Errorf("typed literals are not yet supported")
```

Comment:
What is this one for?

Reply:
Ah I forgot to ask you about it. This one checks for input arguments defined with type prefix like `DATE '2022-02-23'` or `TIME WITH TIME ZONE '12:22:23-0700'`. Any of these input argument with type prefix does not work because we return this error. When I comment it out, it seems it works without any issue.

Reply:
I see, in that case we should remove the commented out portion altogether, rather than leaving it there.